RFC/src/RFC-0301_NamespaceRegistration.md

```diff
@@ -0,0 +1,236 @@
+be grouped under the FQDN by inference. To make this scheme more elaborate, but potentially unnecessary, all such 
+RAIN_IDs could be also be registered on the [base layer] layer, similar to the RAIN scheme.
+
+If the standard Mimblewimble protocol is to be followed, a new output feature can be defined to cater for a RAID tuple 
+`(RAID_ID, PubKey)` that is linked to a specific Unspent Transaction Output ([UTXO]). The `RAID_ID` could be based on 
+a [Base58Check](https://en.bitcoin.it/wiki/Base58Check_encoding) variant applied to `Hash256(PubKey || FQDN)`. If the 
+amount of Tari coins associated with the RAID tuple transaction is burned, `(RAID_ID, PubKey)` will forever be present 
+on the blockchain and can assist with blockchain bloat. On the other hand, if those [Tari coins] are spent back to its 
+owner with a specific time lock, it will be possible to spend and prune that UTXO later on. While that UTXO remains 
+unspent, the RAID tuple will be valid, but when all or part of it is spent, the RAID tuple will disappear from the 
+blockchain. Such a UTXO will thus be "colored" while unspent as it will have different properties to a normal UTXO. It 
+will also be possible to recreate the original RAID tuple by registering it using the original `Hash256(PubKey || FQDN)`.
+
+Thinking about the makeup of the `RAID_ID` it is evident that it can easily be calculated on the fly using the public 
+key and FQDN, both of which values will always be known. The biggest advantage having the RAID tuple on the [base 
+layer] is that of embedded consensus, where it will be validated (as no duplicates can be allowed) and mined before it 
+can be used. However, this comes at the cost of more complex code, a more elaborate asset registration process and 
+higher asset registration fees.
+
+This document explores the creation and use of RAID TXT records to link asset issuer specified domain names with 
+digital assets on the [DAN], without RAID_IDs being registered on the [base layer].
+
+
+
+## Requirements
+
+
+
+### OpenAlias TXT DNS Records
+
+An OpenAlias TXT DNS record [[1]] on a FQDN is a single string and starts with "*oa1:\<name\>*" field followed by a 
+number of key-value pairs. Standard (optional) key-values are: "*recipient_address*"; "*recipient_name*"; 
+"*tx_description*"; "*tx_amount*"; "*tx_payment_id*"; "*address_signature*" and "*checksum*". Additional key-values 
+may also be defined. Only entities with write access to a specific DNS record will be able to create the required TXT 
+DNS record entries.
+
+**Req** - Integration with public DNS records MUST be used to ensure valid ownership of an FQDN that needs to be 
+linked to a [digital asset] on the DAN. 
+
+**Req** - The OpenAlias TXT DNS record MUST be constructed as follows:
+
+| OpenAlias TXT DNS Record Field | OpenAlias TXT DNS Record Data                                |
```

Comment:
Are we having one RAID_ID per asset or one RAID_ID per organisation. 

I am arguing for the latter as managing the former can be needlessly complicated.

Reply:
This RFC is currently promoting neither as it cannot be enforced either way. It is totally up to the asset issuer to provide a valid RAID_ID <> FQDN pair.

So what do you propose, a specific requirement? Example:
"_**Req** - A single linked (`RAID_ID`, FQDN) tuple MAY be associated with multiple digital assets from the same asset issuer._"

Reply:
Yes, I see this as being used with a single RAID_ID per organisation. , Eg BigNeon for all bigneon tickets.

Reply:
Clarification note provided.